packages/odyssey-react/src/components/Radio/RadioButton.module.scss

```diff
@@ -32,33 +32,33 @@
   &.focus {
     + .label {
       &::before {
-        @include outline($focus-ring-primary, 2px);
+        @include outline(var(--ColorFocusOutline), var(--FocusOutlineWidth));
```

Comment:
Not an issue for this PR, as this matches our current standard. However, it feels like a smell to see `PropertyScope` and `ScopeProperty` next to each other.

Reply:
@edburyenegren-okta I think we should make time to revisit our naming system to consider component level variables in finer detail. We've got a good set of challenging examples to point to now including:

* "parts" of a component, infobox icon, banner content etc.
* Functional names vs "descriptive" names (focus is functional, color is descriptive). How to label things like Gutter?
* structuring styles to expose a similar set of theme variables for variants
* variants vs sizes vs other ways to customize vs props and how to name
* Font vs Text labels
* Base vs Default vs Root vs Normal vs Regular etc etc